reconcile/gql_definitions/common/clusters_minimal.gql

```diff
@@ -34,8 +34,16 @@ query ClustersMinimal($name: String) {
         org
         team
       }
-      # ... on ClusterAuthOIDC_v1 {
-      # }
+      ... on ClusterAuthOIDC_v1 {
```

Comment:
nit - cluster minimal :D

Reply:
no need to act. cluster minimal was not minimal already for a long time

Reply:
😂